### DISCUSSION TITLE
Open DevTools with Focused Element Selected in Visual Tree?

### DISCUSSION THREAD

**Discussion by robloo** [AvaloniaUI/Avalonia#7862] [Q&A] [answered]
Is there a pre-existing way to open DevTools with the focused control already selected in the Visual Tree? It's burdensome to click through the hierarchy every time I'm looking for a control.

If not, it would be great to select the focused control when pressing F12 the first time. If DevTools is already open, pressing F12 would automatically update the selection to the latest focused control.

 In Visual Studio it's even easier as it's possible to just click on the the control to highlight it in the live tree instpector.

**Comment by maxkatz6** [comment, marked as answer]
It's not implemented, but makes sense to me.

**Reply by robloo** [reply]
I'm surprised it isn't implemented -- I was expecting it was somehow so I asked it as a question.

I've created a new issue to add this feature. #7869
